### ISSUE TITLE
Documentation for Ubuntu - Java Version and Environment Variable JAVA_HOME might be wrong

### ISSUE THREAD

**Issue by tapwag** [PhoenicisOrg/phoenicis#2656]
### Expected behavior

According to:
https://phoenicisorg.github.io/phoenicis/Developers/build/#ubuntu-18041810-and-linux-mint-19

```
 wget https://download.java.net/java/GA/jdk11/13/GPL/openjdk-11.0.1_linux-x64_bin.tar.gz -O /tmp/openjdk-11+28_linux-x64_bin.tar.gz
 sudo tar xfvz /tmp/openjdk-11+28_linux-x64_bin.tar.gz --directory /usr/lib/jvm
 export JAVA_HOME="/usr/lib/jvm/jdk-11"
```
should work.

### Actual behavior
Unfortunately my system gave me an error message when trying to run 

`mvn clean package`

that JAVA_HOME hadn't been set correctly.

The actual export command of JAVA_HOME variable had to be set to
export JAVA_HOME="/usr/lib/jvm/jdk-11.0.1/"
which includes the 0.1 at the end. 

With this the compile runs fine on my Slackware system.

Also bumping the Ubuntu version to 22.04 LTS might make sense. Is there a git-repository for the documentation where I could contribute?

**Comment by plata** [comment]
Yes, it's https://github.com/PhoenicisOrg/phoenicis/tree/master/docs/_docs.
